data/_ui/tooltips.txt

```diff
@@ -660,6 +660,12 @@ tip "depleted shield delay:"
 tip "shield protection:"
 	`Protection against shield damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in shield damage, while a total value of 11 only results in a 10 percent reduction.`
 
+tip "silent jumps:"
+	`Prevents a ship from making hyperdrive or jump drive sounds when entering or leaving a system.`
+
+tip "silent scans:"
+	`Prevents a ship from generating scan sounds (not even the default scan sound).`
```

Comment:
```suggestion
	`Prevents a ship from generating scan sounds.`
```
I don't think it's necessary, especially that the other one doesn't have this disclaimer.